test/jdk/sun/security/pkcs12/KeytoolOpensslInteropTest.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @bug 8266182
+ * @library /test/lib
+ * @modules java.base/sun.security.pkcs
+ *          java.base/sun.security.util
+ * @summary This is java keytool <-> openssl interop test.
+ *          ParamsTest.java does some operation on pre-generated openssl
+ *          keystores and generate some new JDK keystores under scratch
+ *          directory. The KeytoolOpensslInterop.sh perform some openssl
+ *          command on JDK generated kestores.
+ *
+ *          Prerequisite: Install OpenSSL 1.1.0i or above versions, earlier
+ *          versions may generate different info and test fail.
+ *
+ * @run main/manual/othervm ParamsTest
+ * @run shell/manual KeytoolOpensslInterop.sh
+ */
+
+public class KeytoolOpensslInteropTest {
```

Comment:
This file is not necessary and the shell script can have @run tag instead.

Reply:
Yes, corrected that way. Removed this file and done everything in the shell script.